scheduler/src/cook/mesos/api.clj

```diff
@@ -240,6 +240,11 @@
    (s/optional-key :constraints) [Constraint]
    (s/optional-key :container) Container
    (s/optional-key :executor) (s/enum "cook" "mesos")
+   (s/optional-key :executor-log-level) (s/enum "DEBUG" "ERROR" "INFO" "WARNING")
+   (s/optional-key :executor-max-message-length) PosInt
+   (s/optional-key :progress-output-file) NonEmptyString
+   (s/optional-key :progress-regex-string) NonEmptyString
+   (s/optional-key :progress-sample-interval-ms) PosInt
```

Comment:
What's the use case for letting users modify the sample interval?

Reply:
If users know their rate of output is going to be really slow, they can set `progress-sample-interval-ms` to a higher value than the default.